.pre-commit-config.yaml

```diff
@@ -56,3 +56,12 @@ repos:
         args:
           - --fix=lf
       - id: trailing-whitespace
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: cb934cfd6d2eea1aed7b20e4e9e6339d5952322c # 0.28.0
+    hooks:
+      # Schemas taken from https://www.schemastore.org/json/
+      - id: check-jsonschema
+        name: "Validate GitHub issue templates"
+        files: ^\.github/ISSUE_TEMPLATE/.*\.yml$
+        exclude: ^\.github/ISSUE_TEMPLATE/config\.yml$
+        args: ["--verbose", "--schemafile", "schemas/github-issue-forms.json"]
```

Comment:
Why can you add this to the `.pre-commit-config.yaml` of the template? Then they both get validation

Reply:
> Unfortunately it's not possible to validate the package template files as they contain invalid characters for a string entry ({ and }).

Reply:
Ah sorry, just realised what you meant! Good shout, I will do that.

Reply:
👍 done

Reply:
I did wonder if you'd missed something